src/api_tracker.py

```diff
@@ -0,0 +1,68 @@
+import os
+import requests
+from ckan.common import config
+import ckan.plugins.toolkit as tk
+import hashlib
+import logging
+import re
+
+
+log = logging.getLogger(__name__)
+
+def ga_api_tracker(resource_id):
+    try:
+        tid = config.get('googleanalytics.id', None)
+        endpoint = config.get('googleanalytics.endpoint', None)
+        client_id = hashlib.md5(tk.c.user).hexdigest()
+        payload = {
+            'v': 1, # Version.
+            'tid': tid, # Tracking ID
+            'cid': client_id, # Anonymous Client ID
+            't': 'event', # Hit type (event, pageview, etc)
+            'ec': 'Resource', # Event Category
+            'ea': 'API', # Event Action
+            'el': resource_id # Event label
+        }
+        log.warning('sending payload to GA: {}'.format(payload))
+        r = requests.post(endpoint, data=payload, headers={'User-Agent': 'User Agent 1.0'})
+        if r.ok:
+            log.info('ga for search api sent')
+        else:
+            log.info('ga for search api failed')
+    except Exception as e:
+        log.error(str(e))
+
+def ga_search_sql_api_tracker(sql):
```

Comment:
i think we want to track both the table if poss *and* save the sql statement on that event if we can ... and still store this if we can't get the table.

Reply:
I need to combine these two parameters(table + SQL statement) in `Event label` and it looks not readable inreport I think...
I can divide them by `:` or `\`  or maybe some other symbol?
Probably we should store the SQL statement only if the table name was not retrieved from the statement fro any reason?
WDYT?
![Screenshot from 2020-07-30 14-02-24](https://user-images.githubusercontent.com/61929774/88916767-53f52900-d26f-11ea-886b-c35e291cba73.png)

Reply:
looks good.